src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcAddToAList.scala.html

```diff
@@ -0,0 +1,104 @@
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.button.Button
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.fieldset.{Fieldset, Legend}
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.{RadioItem, Radios}
+@import uk.gov.hmrc.hmrcfrontend.views.helpers.MessageHelpers.{messagesForLanguage, pluralMessageKey}
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.AddToAList
+
+@this(govukRadios: GovukRadios, govukButton: GovukButton)
+
+@(params: AddToAList)(implicit messages: Messages, messagesApi: MessagesApi)
+@import params._
+
+@defining(messagesForLanguage(language)) { messages =>
+
+    <h1 class="govuk-heading-xl">
+        @messages(
+            pluralMessageKey("addToAList.heading", itemList.length),
+            itemList.length,
+            itemType.map { itemType =>
+                itemList.length match {
+                    case 1 => itemType.singular
+                    case _ => itemType.plural
+                }
+            }.getOrElse {
+                messages(pluralMessageKey("addToAList.defaultItemType", itemList.length))
+            }
+        )
+    </h1>
+
+    <div class="govuk-form-group">
+        <dl class="hmrc-add-to-a-list hmrc-add-to-a-list--@itemSize.suffix">
+        @for(item <- itemList) {
+            <div class="hmrc-add-to-a-list__contents">
+                <dt class="hmrc-add-to-a-list__identifier hmrc-add-to-a-list__identifier--light">
+                @item.name
+                </dt>
+                <dd class="hmrc-add-to-a-list__change">
+                    <a class="govuk-link" href="@item.changeUrl">
+                        <span aria-hidden="true">@messages("addToAList.change.displayed")</span>
+                        <span class="govuk-visually-hidden">@messages("addToAList.change.announced", item.name)</span>
+                    </a>
+                </dd>
+                <dd class="hmrc-add-to-a-list__remove">
+                    <a class="govuk-link" href="@item.removeUrl">
+                        <span aria-hidden="true">@messages("addToAList.remove.displayed")</span>
+                        <span class="govuk-visually-hidden">@messages("addToAList.remove.announced", item.name)</span>
+                    </a>
+                </dd>
+            </div>
+        }
+        </dl>
+    </div>
+    <form method="post" action="@formAction">
+        @govukRadios(Radios(
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                    content = Text(messages(
+                        "addToAList.addAnother",
+                        itemType.map { itemType => itemType.singular }.getOrElse {
+                            messages(pluralMessageKey("addToAList.defaultItemType", 1))
+                        }
+                    )),
+                    classes = "govuk-fieldset__legend govuk-fieldset__legend--m",
```

Comment:
- [ ] going to raise a bug seperately, leads to element having 2 `govuk-fieldset__legend` classes

Reply:
yup, sorry I wasn't clear I'll edit it, I meant raise it against hmrc-frontend - I'm just matching the nunjucks template here